packages/server/src/training/index.ts

```diff
@@ -40,92 +48,112 @@ export default class Trainer {
     });
   }
 
-  start(id: string, hyperparameters: unknown): void {
-    this.projects[id] = {
-      training_container: null,
-      metrics_container: null,
-      metrics: null
-    };
+  async start(id: string, hyperparameters: unknown): Promise<string> {
```

Comment:
The hyper parameters we are passing in need to be typed. We are also using this argument as a catch all right now (for things more than hyper parameters). We should pass other variables in as other variables.